frontend/src/common/queries/cellGuide.ts

```diff
@@ -229,115 +272,128 @@ export const useCanonicalMarkers = (
   );
 };
 
-/* ========== CL description ========== */
-export const USE_CL_DESCRIPTION_QUERY = {
-  entities: [ENTITIES.CELL_GUIDE_CL_DESCRIPTION],
-  id: "cell-guide-cl-description-query",
+/* ========== latest snapshot identifier ========== */
+export const USE_LATEST_SNAPSHOT_IDENTIFIER_QUERY = {
+  entities: [ENTITIES.CELL_GUIDE_LATEST_SNAPSHOT_IDENTIFIER],
+  id: "cell-guide-latest-snapshot-identifier-query",
 };
 
-export type ClDescriptionQueryResponse = string;
+export type LatestSnapshotIdentifierQueryResponse = string;
 
-export const useClDescription = (entityId: string): UseQueryResult<string> => {
-  return useCellGuideQuery<string>(TYPES.CL_DESCRIPTION, entityId);
-};
-
-/* ========== UBERON description ========== */
-export const USE_UBERON_DESCRIPTION_QUERY = {
-  entities: [ENTITIES.CELL_GUIDE_UBERON_DESCRIPTION],
-  id: "cell-guide-uberon-description-query",
-};
-
-export type UberonDescriptionQueryResponse = string;
-
-export const useUberonDescription = (
-  entityId: string
-): UseQueryResult<string> => {
-  return useCellGuideQuery<string>(TYPES.UBERON_DESCRIPTION, entityId);
+export const useLatestSnapshotIdentifier = (): UseQueryResult<string> => {
+  return useCellGuideQuery<string>(TYPES.LATEST_SNAPSHOT_IDENTIFIER);
 };
 
 /* ========== description ========== */
-export const USE_DESCRIPTION_QUERY = {
-  entities: [ENTITIES.CELL_GUIDE_DESCRIPTION],
-  id: "cell-guide-description-query",
+export const USE_GPT_DESCRIPTION_QUERY = {
+  entities: [ENTITIES.CELL_GUIDE_GPT_DESCRIPTION],
+  id: "cell-guide-gpt-description-query",
 };
 
-export type DescriptionQueryResponse = string;
+export type GptDescriptionQueryResponse = string;
 
-export const useDescription = (entityId: string): UseQueryResult<string> => {
-  return useCellGuideQuery<string>(TYPES.DESCRIPTION, entityId);
+export const useGptDescription = (entityId: string): UseQueryResult<string> => {
+  return useCellGuideQuery<string>(TYPES.GPT_DESCRIPTION, entityId);
 };
 
-/* ========== cell_guide_cards ========== */
-export const USE_CELL_GUIDE_QUERY = {
-  entities: [ENTITIES.CELL_GUIDE_CELL_GUIDE],
-  id: "cell-guide-query",
+/* ========== SEO description ========== */
+export const USE_GPT_SEO_DESCRIPTION_QUERY = {
+  entities: [ENTITIES.CELL_GUIDE_GPT_SEO_DESCRIPTION],
+  id: "cell-guide-gpt-seo-description-query",
 };
 
-interface CellGuideQueryResponseEntry {
-  id: string;
-  label: string;
-  synonyms?: string[];
+interface GptSeoDescriptionQueryResponse {
+  name: string;
+  description: string;
 }
 
-export type CellGuideQueryResponse = CellGuideQueryResponseEntry[];
+export const fetchGptSeoDescription = async (
```

Comment:
React Query seems to support SSR, should we look into this?

https://tanstack.com/query/v4/docs/react/guides/ssr#using-initialdata

This way React Query will cache the API response for us in the frontend server

Reply:
Yeah, but the responses on the server side still won't be cached, right? Using `initialData`, for example, would just make it so that the client can use the result of the server. This is something we should do as well. However, I'm also concerned about the fact that the server itself isn't doing any form of caching.